react/__tests__/index.test.tsx

```diff
@@ -732,4 +732,27 @@ describe('GA4 events', () => {
       })
     })
   })
+  describe('share', () => {
```

Comment:
Nitpick 💅: just an empty line to separate different tests.

```suggestion

  describe('share', () => {
```